extensions/machine-learning-services/src/test/packageManagement/packageManager.test.ts

```diff
@@ -32,13 +32,26 @@ describe('Package Manager', () => {
 		testContext.apiWrapper.verify(x => x.executeCommand(TypeMoq.It.isAny(), TypeMoq.It.isAny(), TypeMoq.It.isAny(), TypeMoq.It.isAny()), TypeMoq.Times.once());
 	});
 
+	it('Manage Package command Should execute the command if r installed', async function (): Promise<void> {
```

Comment:
>r [](start = 58, length = 1)

What about if just python is installed? Or neither?

Reply:
I have a test for those scenarios as well

---
In reply to: [366618808](https://github.com/microsoft/azuredatastudio/pull/8870#discussion_r366618808) [](ancestors = 366618808)